### ISSUE TITLE
invalid token at map.convert()

### ISSUE THREAD

**Issue by mhmxs** [ballerina-platform/ballerina-lang#12921]
**Description:**
I tried to execute example, but the compilation fails on my machine.

**Steps to reproduce:**
Create file: [file](https://github.com/ballerina-platform/ballerina-lang/blob/90e7560b42ad249c87c95f5b2f1d1ff88d845455/examples/json-record-map-conversion/json_record_map_conversion.bal) locally.
Than:
```
ballerina build
Compiling source
    x.bal
error: .::x.bal:36:9: invalid token 'movieMap'
error: .::x.bal:36:23: invalid token '.'
error: .::x.bal:37:9: invalid token 'writerMap'
```

**Affected Versions:**
Ballerina 0.990.0

**OS, DB, other environment details and versions:**
macOs Mojave 10.14.1

**Comment by MaryamZi** [comment]
@mhmxs there was an issue with this particular example which we've fixed for the upcoming release (0.990.1). This will be reflected on ballerina.io soon.

The updated example is available at https://github.com/ballerina-platform/ballerina-lang/blob/v0.990.1/examples/json-record-map-conversion/json_record_map_conversion.bal.

Please let us know if you are facing any issues with the updated example. Sorry about the inconvenience!

**Comment by mhmxs** [comment]
Thank a lot, solved my issue.
